src/Education/XamlGenerator/XamlGeneratorHelper.cs

```diff
@@ -123,24 +214,29 @@ private void WriteSubTitleElement_Severity(IRuleInfo ruleInfo)
                 WriteSubTitleElementWithImage(imageInfo);
             }
 
-            private void WriteSubTitleElementWithImage(SubTitleImageInfo imageInfo)
-            {
-                writer.WriteStartElement("Span");
-                writer.ApplyStyleToElement(StyleResourceNames.SubtitleElement_Span);
+            #endregion
 
-                if (imageInfo.ImageResourceName != null)
-                {
-                    writer.WriteStartElement("InlineUIContainer");
-                    writer.WriteStartElement("Image");
-                    writer.ApplyStyleToElement(StyleResourceNames.SubtitleElement_Image);
-                    writer.WriteAttributeString("Source", $"{{DynamicResource {imageInfo.ImageResourceName}}}");
-                    writer.WriteEndElement(); // Image
-                    writer.WriteEndElement(); // InlineUIContainer
-                }
+            #region Common
 
-                writer.WriteString(imageInfo.DisplayText);
-                writer.WriteEndElement(); // Span
-            }
+            private void WriteSubTitleElementWithImage(SubTitleImageInfo imageInfo, bool useCctStyle = false) =>
```

Comment:
Minor: it might be a bit neater if the caller passed in the style to use so this method doesn't need to special case CCT.

Reply:
It's used 2 times for 2 different styles, would be too inconvenient